lib/rucio/core/transfer.py

```diff
@@ -702,6 +781,271 @@ def __rewrite_dest_url(dest_url, dest_sign_url, dest_scheme):
     return dest_url
 
 
+def __prepare_source_dest_config(ctx, request_id, src_rse_id, dest_rse_id, source_scheme, dest_scheme, dict_attributes, activity,
```

Comment:
Add a short description what this does.

Reply:
merged this function into the calling context. I feel it is better this way.